apps/journeys-admin/src/components/FramePortal/FramePortal.tsx

```diff
@@ -0,0 +1,94 @@
+import { createPortal } from 'react-dom'
+import createCache from '@emotion/cache'
+import { CacheProvider } from '@emotion/react'
+import {
+  DetailedHTMLProps,
+  IframeHTMLAttributes,
+  memo,
+  ReactElement,
+  ReactNode,
+  useEffect,
+  useMemo,
+  useReducer,
+  useRef
+} from 'react'
+import { styled } from '@mui/material'
+import { StylesProvider } from '@mui/styles'
+
+interface ContentProps {
+  children: ReactNode
+  document: Document
+}
+
+function Content({ children, document }: ContentProps): ReactElement {
+  const cache = useMemo(
+    () =>
+      createCache({
+        key: 'iframe',
+        container: document.head,
+        prepend: true
+      }),
+    [document]
+  )
+
+  return (
+    <StylesProvider>
+      <CacheProvider value={cache}>{children}</CacheProvider>
+    </StylesProvider>
+  )
+}
+
+const StyledFrame = styled('iframe')(() => ({
+  border: 0
+}))
+
+interface FrameProps
+  extends Omit<
+    DetailedHTMLProps<
+      IframeHTMLAttributes<HTMLIFrameElement>,
+      HTMLIFrameElement
+    >,
+    'css'
+  > {
+  children: ReactNode
+}
+
+export const FramePortal = memo(function FramePortal({
+  children,
+  ...other
```

Comment:
```suggestion
  ...props
```
Shall we stick to React terminology?

Reply:
is this the norm in react? I thought props was only used when it contained all the props. In react docs i've seen other and rest as the name of the remaining props.

Reply:
going with rest as I have seen that elsewhere.

Reply:
I've mainly just seen props